packages/strapi-plugin-content-type-builder/controllers/ContentTypeBuilder.js

```diff
@@ -47,11 +48,13 @@ module.exports = {
       return ctx.badRequest(null, [{ messages: attributesErrors }]);
     }
 
+    const _description = escapeNewlines(description, '\\n');
```

Comment:
Do we need to store this value in a variable. Just used one time in this function.

Reply:
I think it's clearer and easier to read an explicit declaration than a direct use of a function, but I can change it if you want. That's not a problem.